scripts/convert-qpp-to-measures.js

```diff
@@ -118,7 +118,7 @@ process.stdin.on('end', () => {
  *    emsr_id               | eMeasureId
  *    nqf_emsr_num          | nqfEMeasureId
  *    nqf_num               | nqfId
- *    qlty_id               | qualityId
+ *    qlty_id               |
```

Comment:
Hmm maybe this should be removed altogether - are there other fields we don't use?